mitodl/mitxpro
# Optimized bulk purchase page

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #1626 

#### What's this PR do?
Creates an option in the product API endpoint that limits the depth of results and configures the B2B bulk purchase page to use that endpoint

#### How should this be manually tested?
Go to the B2B bulk purchase page (`/ecommerce/bulk/`). Complete a purchase with a program and course run, with and without a coupon applied, and make sure validation works as expected
